### DISCUSSION TITLE
官方是否有主题配置器的全json字段文档

### DISCUSSION THREAD

**Discussion by KahrLu** [tusen-ai/naive-ui#3019] [Ideas]
如果没有的话希望提供一下谢谢，现在官网右下角有一个主题编辑器，勾选几个值导出的话只有那几个字段的json。
如果能有个全json像字典一样可以查看的话，方便开发者使用

**Comment by 07akioni** [comment]
未来可能会提供，目前建议通过 ts 类型安排一下
